File: api_platform/api-platorm-backend/admin/model/api_category.py
from common.db.base_model import Base,Mapped,mapped_column,BigInteger,Integer,String,Text

class APICategory(Base):
    __tablename__ = "api_category"

    parent_id:Mapped[int] = mapped_column("parent_id",BigInteger,nullable=False)
    category_name:Mapped[str] = mapped_column("category_name",String,nullable=False)
    sort:Mapped[int] = mapped_column("sort",Integer,nullable=False)
    ancestors:Mapped[str] = mapped_column("ancestors",Text,nullable=False)
    creator_id:Mapped[int] = mapped_column("creator_id",BigInteger,nullable=False)
    

File: api_platform/api-platorm-backend/user/clientuser/src/main/java/com/whs/apiplatform/api/domain/APICategory.java
package com.whs.apiplatform.api.domain;

import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
import com.whs.apiplatform.common.model.BaseModel;
import lombok.Data;

@Data
public class APICategory extends BaseModel {

    @JsonSerialize(using = ToStringSerializer.class)
    private Long parentId;
    private String categoryName;
    private Integer sort;
    private String ancestors;
    @JsonSerialize(using = ToStringSerializer.class)
    private Long creatorId;

}
